### PR TITLE
EW member photo

### LINKED ISSUE #222: Add member image for Edward Wu

### PR DESCRIPTION
**Pull request by ctc-devops**
closes #222

### CODE REVIEW

File: website/src/app/views/about/members.js [hunk 1]
```diff
@@ -135,6 +135,7 @@ import kyledoliente from '../../../images/about/memberImages/kyledoliente.jpg';
 import kanu from '../../../images/about/memberImages/kanu.jpeg';
 import mazenghosn from '../../../images/about/memberImages/mazenghosn.jpg';
 import hassan from '../../../images/about/memberImages/hassan.jpg';
+import EdWu from '../../../images/about/memberImages/EdWu.jpeg';
```
Comment (Vakhshoori101) [inline website/src/app/views/about/members.js:138]: rename "EdWu" to be your first name and last name ("edwardwu")

**Review by Vakhshoori101 — CHANGES_REQUESTED** [review]
Everything looks good. Address the one comment and re-upload your picture with dimensions of 260x260 pls

File: website/src/app/views/about/members.js (hunk 1 above)
Comment (Edwu29) [reply website/src/app/views/about/members.js]: 👍

**Review by Vakhshoori101 — APPROVED** [review]
LGTM

**Review by Vakhshoori101 — DISMISSED** [review]
LGTM

File: website/src/app/views/about/members.js [hunk 2]
```diff
@@ -967,6 +968,12 @@ const members = {
       linkedinURL: 'https://www.linkedin.com/in/kyle-doliente-877133225/',
       imageURL: kyledoliente,
     },
+    {
+      name: 'Edward Wu',
```
Comment (Vakhshoori101) [inline website/src/app/views/about/members.js:972]: Sry just realized that your member info should be in the developer group. So move it into the developer group sorted by your first name. Then I can merge!

Comment (Edwu29) [reply website/src/app/views/about/members.js]: Oh gotcha, I didn't realize that either

**Review by Vakhshoori101 — APPROVED** [review]
LGTM

**Pull request merged**